apps/web/pages/apps/[slug]/index.tsx

```diff
@@ -53,15 +53,15 @@ export const getStaticPaths: GetStaticPaths<{ slug: string }> = async () => {
 
   return {
     paths,
-    fallback: false,
+    fallback: "blocking",
```

Comment:
what does this do?

Reply:
all the paths returned by `getStaticPaths` are lower cased. with the fallback false, slug with the title case throws an 404 error. while fallback is blocking, it runs `getStaticProps` and check if it's matches any db record.